libsel4/sel4_arch_include/aarch32/sel4/sel4_arch/constants.h

```diff
@@ -117,6 +117,7 @@ typedef enum {
     seL4_VCPUReg_CNTV_CVALlow,
     seL4_VCPUReg_CNTVOFFhigh,
     seL4_VCPUReg_CNTVOFFlow,
+    seL4_VCPUReg_CNTKCTL,
```

Comment:
This will probably also need an update in the CHANGES file

Reply:
Also discovered that `vcpu_padding` in `struct vcpu` needs to be removed, otherwise the struct is no longer packed